examples/ale_classification.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<ul><li>Forgot a space before the bullets start (bullets don't show in rendered version)</li><li>_Whils_ -&gt; W<em>hile</em></li><li>Second bullet point does not seem to be true for the GBM </li><li>Both models do perfectly on test data, so we cannot talk simply about an "increase in discriminative power"; we can say that the models use different features to discern between different classes. An interesting experiment would be to see what would happen if we excluded petal width data, refit the models and re-assessed them. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>